src/ProjectTemplates/Web.ProjectTemplates/content/WebApiAot-CSharp/.template.config/localize/templatestrings.de.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
   "author": "Microsoft",
-  "name": "ASP.NET Core-API",
+  "name": "ASP.NET Core Web API (native AOT)",
```

Comment:
How do the localized messages get updated when we update the English/invariant values? I see some of these have parens `(native AOT)`, where some of the languages don't have parens.

For example the spanish one says "API principal de ASP.NET"

Reply:
Yeah I spoke to @captainsafia about this and apparently it happens as part of build-ops?